HousingFinanceInterimApi/V1/Infrastructure/IDatabaseContext.cs

```diff
@@ -63,5 +64,31 @@ public interface IDatabaseContext
         Task<List<string[]>> GetItemisedTransactionsByYearAndTransactionTypeAsync(int year, string transactionType);
         Task<List<string[]>> GetHousingBenefitAcademyByYear(int year);
         Task<IList<string[]>> GetReportAccountBalance(DateTime reportDate, string rentGroup);
+
+        public DbSet<BatchLog> BatchLogs { get; set; }
```

Comment:
These weren't in the interface and so didn't allow access in the tests - added these for current and future convenience